Open WorldCat.js

```diff
@@ -67,7 +67,9 @@ function getFirstContextObj(doc) {
 }
 
 function detectWeb(doc, url) {
-	var results = getSearchResults(doc);
+	//distinguish from Worldcat Discovery
+	if (ZU.xpath(doc, '/html/body[@id="worldcat"]').length >0){
```

Comment:
Nit #1: I think you should be able to just do `doc.body.id == "worldcat"`

Not #2: indents (unless it just doesn't show up here in the diff)

Other than that, fine by me